src/propmcwf.jl

```diff
@@ -0,0 +1,139 @@
+# TODO : QuBase.jl - WIP : complex and expectation.
+
+@doc """
+Quantum Monte-Carlo Wave Function Method
+Input Parameters :
+`eps`     : random number used in comparison of jtol and norm of the state drawn
+`options` : Dictionary to set the solver used to proagate the state generated using draw.
+
+Step Propagation using Quantum Monte-Carlo Wave Function Method.
+""" ->
+type QuMCWF <: QuPropagatorMethod
+    eps::Float64
+    options::Dict{Symbol, Any}
+end
+
+QuMCWF(options::Dict) = QuMCWF(rand(), options)
```

Comment:
You can fold these two constructors into one by using a default value for `options` (`options::Dict=Dict()`).